### DISCUSSION TITLE
The Sintel video downloads alright, but never read in video tag

### DISCUSSION THREAD

**Discussion by jocimarlopes** [webtorrent/webtorrent#2700] [Q&A]
If I open this link, i'll download the video, but when I add the same link/video at DOM, in a video tag, not works...
The response is that:

GET http://localhost:8100/assets/js/webtorrent/08ada5a7a6183aae1e09d831df6748d566095a10/Sintel.mp4 net::ERR_ABORTED 404 (Not Found)

Someone help me with that?

**Comment by SilentBot1** [comment]
Hey @jocimarlopes,

I believe this possibly relates to this issue: https://github.com/webtorrent/webtorrent/issues/2374

Are you able to confirm if you're using a library such as React which possibly is possibly patching the global fetch function?
